test_data/asm/single_operation.asm

```diff
@@ -14,4 +14,7 @@ machine Main with degree: 8 {
     SingleOperation m;
 
     link => m.nothing();
+
+    col witness w;
+    w = w * w;
```

Comment:
Somehow, without this, the columns generated by the bus interaction end up in the `std::well_known` namespace, which doesn't have a degree... I thought it's not worth fixing it, because a machine without a column is very unrealistic.